images/installer/root/usr/local/bin/entrypoint-gcp

```diff
@@ -1,51 +0,0 @@
-#!/bin/bash
```

Comment:
Can you make this a wrapper that calls /usr/local/bin/entrypoint TYPE=gcp so that CI jobs don't break?

Reply:
@sdodson, yeah or we can leave the entrypoint-gcp until we arrive at a convergence point.